firmware/shared/src/hw/hw_spi.h

```diff
@@ -37,7 +37,7 @@ void hw_spi_setNssHigh(const SpiInterface *spi);
  * @return True if data is transmitted and received successfully. Else return
  * false.
  */
-bool hw_spi_transmitThenReceive(
+bool hw_spi_transmitAndReceive(
```

Comment:
really hope this does not break random BMS code

Reply:
Ok I don't even remember changing this so I'll change it back